dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/passwordreset/PasswordResetServiceBean.java

```diff
@@ -90,12 +96,36 @@ public PasswordResetInitResponse requestPasswordReset( BuiltinUser aUser, boolea
         
     }
 
+    private void updateTokenExpirationTime(BuiltinUser aUser) {
+
+        long ONE_MINUTE_IN_MILLISECONDS = 60000;
+        long defaultInMilliseconds = systemConfig.getMinutesUntilPasswordResetTokenExpires() * ONE_MINUTE_IN_MILLISECONDS;
+
+
+        List<PasswordResetData> passwordResetDatas = findPasswordResetDataByDataverseUser(aUser);
+        if(passwordResetDatas.size() > 1) {
+            logger.severe(aUser + " has more than one Reset Password Token");
```

Comment:
Imo logs are unnecessary here, imagine 250k concurrent user flooding logs with this.

Reply:
passwordResetDatas.size() != 1 at this point should be VERY uncommon. If this happens it indicates a deep problem within application - that's why i've decided to add logging here.

Reply:
I understand, idea is right but if it is 'exceptional' state then exception should be thrown which would be much more informative(stack trace). And if it is just user error IMO the Either<error,success> should be returned and then at the end, you can log the error and also use jsf to show the error.